VERSION

```diff
@@ -1 +1 @@
-0.0.1
+0.0.2
```

Comment:
I think minor version should start from 1. I.e. the smallest initial version is `0.1.0`
(https://semver.org/)